doc/api/process.md

```diff
@@ -1342,6 +1342,12 @@ The `process.getgroups()` method returns an array with the supplementary group
 IDs. POSIX leaves it unspecified if the effective group ID is included but
 Node.js ensures it always is.
 
+```js
+if (process.getgroups) {
```

Comment:
Is this check because the function doesn't exist on Windows? If so, the sample code should probably go below the sentence that indicates this.

Reply:
On further review, never mind the "move it below the other sentence" comment above. This conforms with other examples in the doc and it hasn't caused problems yet, so this is probably just fine.